Lib/urllib/request.py

```diff
@@ -163,18 +163,10 @@ def urlopen(url, data=None, timeout=socket._GLOBAL_DEFAULT_TIMEOUT,
 
     The *cadefault* parameter is ignored.
 
-    This function always returns an object which can work as a context
-    manager and has methods such as
 
-    * geturl() - return the URL of the resource retrieved, commonly used to
-      determine if a redirect was followed
-
-    * info() - return the meta-information of the page, such as headers, in the
-      form of an email.message_from_string() instance (see Quick Reference to
-      HTTP Headers)
-
-    * getcode() - return the HTTP status code of the response.  Raises URLError
-      on errors.
+   This function always returns an object which can work as a
+   :term:`context manager` and has the properties `url`, `headers`, and `status`.
+   See `urllib.response.addinfourl` for more detail on these properties.
```

Comment:
Need a role to make this a link!

```suggestion
   See :class:`urllib.response.addinfourl` for more detail on these properties.
```

Reply:
@merwok This is a docstring.

Reply:
I missed that!  Indeed, markup should be removed (not `*` but all rst roles and directives).